src/resources/plsPlusAddress/script/defaultValue.js

```diff
@@ -0,0 +1,61 @@
+const bodyElem = document.querySelector('body');
+const plsPlusInit = bodyElem.getAttribute('data-plsPlusInit');
+
+function getFranchise(myFranchise) {
+  if (myFranchise === "auto") {
+    const host = location.host;
+    if (host.indexOf("forgov") !== -1) {
+      return "forgov";
+    } else if (host.indexOf("tmr") !== -1) {
+      return "tmr";
+    } else if (host.indexOf("covid19") !== -1) {
+      return "covid19";
+    } else if (host.indexOf("health") !== -1) {
+      return "health";
+    // https://www.resource.qld.gov.au/ is not found
+    // } else if (host.indexOf("resource") !== -1) {
+    //   return "resource";
+    } else {
+      return "qldonline";
```

Comment:
uncertain about using this as default key. maybe the  limited or form.io key.

Reply:
per vani's suggestion, i'd go with not having a default and log a message to console if missing property or can't find.